package.json

```diff
@@ -63,7 +64,6 @@
     "@types/node": "^18.7.1",
     "@typescript-eslint/eslint-plugin": "^5.40.0",
     "@typescript-eslint/parser": "^5.40.0",
-    "@venusprotocol/isolated-pools": "^2.0.0",
     "@venusprotocol/oracle": "1.8.0-dev.1",
```

Comment:
I wonder if oracle dev dependency should be a real dependency alongside others

Reply:
Yeah, I mean other contract packages (like OZ contracts, governance, protocol reserve) are listed in `dependencies` section, while oracle is in `devDependencies`. I'm curious if there's any reason behind that, or it's just happened so during the code evolution